### PR TITLE
lv-tool: Upgrade default actor plugin to "lv_gltest" if available

### PR DESCRIPTION
**Pull request by hartwork**
This is a foward port of 0.4.x commit e4a6cf5b60771b8cffa90834c0dd967b006bd3b1 from pull request #150 to `master`.

### CODE REVIEW

File: libvisual/libvisual/lv_actor.h [hunk 1]
```diff
@@ -52,6 +52,15 @@ namespace LV
...
+       *
+       * @param name Name of plugin to load
+       *
+       * @return True if an actor by that name is available, else false
+       */
+      static bool available (std::string const& name);
+
       /**
        * Creates a new Actor with a plugin of a given name.
        *
```
Comment (kaixiong) [inline libvisual/libvisual/lv_actor.h:66]: Having this is a good idea. Can we move this into a separate patch and add the method to `LV::Morph` and `LV::Input` as well?

**Review by kaixiong — CHANGES_REQUESTED** [review]

File: libvisual/libvisual/lv_actor.h (hunk 1 above)
Comment (hartwork) [reply libvisual/libvisual/lv_actor.h:66]: @kaixiong done :+1:

**Review by kaixiong — CHANGES_REQUESTED** [review]
@hartwork, thanks. Can you insert a space between the `available()` and its argument list, like this:
```cpp
if (!available (name)) {
```

**Comment by hartwork** [comment]
> @hartwork, thanks. Can you insert a space between the `available()` and its argument list, like this:
> 
> ```c++
> if (!available (name)) {
> ```

@kaixiong done

**Review by kaixiong — APPROVED** [review]
@hartwork, all good to go.
